integration_tests/schemaops/utestschema/helper.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2023 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utestschema
+
+import (
+	"testing"
+
+	"github.com/openconfig/ygot/ygot"
+)
+
+func GetOrderedMap(t *testing.T) *Ctestschema_OrderedLists_OrderedList_OrderedMap {
```

Comment:
godoc for funcs in this file
nit: GetSampleOrderedMap or something to indicate that this returns an ordered with by populated values

Reply:
Done. I'll also move these to an internal package in a subsequent PR.